components/html-block/test/html-block.visual-diff.js

```diff
@@ -34,7 +34,9 @@ describe('d2l-html-block', () => {
 		{ name: 'math (inline)', selector: '#math-inline' },
 		{ name: 'code (block)', selector: '#code-block' },
 		{ name: 'code (inline)', selector: '#code-inline' },
-		{ name: 'math (block) and code (block)', selector: '#math-block-and-code-block' }
+		{ name: 'math (block) and code (block)', selector: '#math-block-and-code-block' },
+		{ name: 'using html property', selector: '#html-property' },
+		{ name: 'using both html property and slot', selector: '#html-property-and-slot' }
```

Comment:
I didn't see a point adding specific unit tests for these cases for now, since eventually the idea is that we'll only have two cases: property (deferred) and slotted (non-deferred), so those tests are going to need a rewrite anyways. In the meantime, I think these visual diff tests will do.